lib/Test/Nginx/Util.pm

```diff
@@ -2364,6 +2373,23 @@ retry:
             #warn "pid file not found";
         }
     }
+
+shutdown_error_log:
+    if (defined $block->shutdown_error_log) {
+        if ($blocks_left != 0 && $UseHup) {
+            return
```

Comment:
> Also, how do you handle the case when TEST_NGINX_NO_CLEAN is specified? And how do you handle the last test block using --- shutdown_error_log?

throw a warning message when `TEST_NGINX_NO_CLEAN` is specified and do skip now.